doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
+      * Branches in the nodejs/node repository are only for release lines
     * [See "Updating Node.js from Upstream"](./onboarding-extras.md#updating-nodejs-from-upstream)
-    * make new branches for all commits you make!
+    * Make new branches for all commits you make!
 
-  * notifications:
-    * use [https://github.com/notifications](https://github.com/notifications) or set up email
-    * watching the main repo will flood your inbox, so be prepared
+  * Notifications:
+    * Use [https://github.com/notifications](https://github.com/notifications) or set up email
+    * Watching the main repo will flood your inbox (several hundred notifications on typical weekdays), so be prepared
 
   * `#node-dev` on [webchat.freenode.net](https://webchat.freenode.net/) is the best place to interact with the CTC / other collaborators
+    * If there are any questions after the session, a good place to ask is there!
+    * Presence is not mandatory, but please drop a note there if force-pushing to `master`
 
 
 ## Project goals & values
 
-  * collaborators are effectively part owners
-    * the project has the goals of its contributors
+  * Collaborators are effectively the collective owners of the project
```

Comment:
Take-it-or-leave-it nit: While we're in here, let's drop `effectively`. It introduces more questions than it answers. Collectively, the Collaborators are the owners of the project. Saying `effectively` just signals that we don't actually mean it.